src/common/database.c

```diff
@@ -1523,6 +1523,15 @@ static int _upgrade_library_schema_step(dt_database_t *db, int version)
 
     new_version = 24;
   }
+  else if(version == 24)
+  {
+    sqlite3_exec(db->handle, "BEGIN TRANSACTION", NULL, NULL, NULL);
```

Comment:
Not a big deal, but a transaction for a single statement is overzealous :)

Reply:
TBH version upgrades with per-version transaction is IMO good practice ;) It's just 2 lines more ;)

Reply:
I've already made the change ... :(

Reply:
Well, let me know ... that's easy :)